pkg/apply/task/apply_task.go

```diff
@@ -116,6 +127,22 @@ func (a *ApplyTask) Start(taskContext *taskrunner.TaskContext) {
 					object.UnstructuredToObjMeta(obj), event.Unchanged, applyerror.NewUnknownTypeError(err))
 				continue
 			}
+
+			clusterObj, err := getClusterObj(dynamic, info)
+			if err != nil {
+				if !apierrors.IsNotFound(err) {
+					fmt.Println("I'm here in line 134")
```

Comment:
We probably want to remove this line.

Reply:
Thanks for catching this : ).  Will remove it.